float64[] cmd_vel
float64 duration
---

bool success